src/syntax-highlight/language-ext.js

```diff
@@ -63,5 +65,9 @@ module.exports = {
     '.wxs': 'language-markup',
     '.xaml': 'language-markup',
     '.xsd': 'language-markup',
-    '.yml': 'language-yaml'
+    '.yml': 'language-yaml',
+
+    // Universally named files
+    'jenkinsfile': 'language-groovy',
+    'vagrantfile': 'language-ruby'
 };
```

Comment:
Can you leave them without the quotes? Prettier will end up removing them as you said for the next developer that changes this file, and it will show up in the diff for that change, which would be unrelated

Reply:
Thanks for the tip. I had to switch back to Atom to get it working - will recheck my VS Code settings later.  Sweet stuff, learnt a few things doing this. Thanks again. I'll throw a few PRs for highlighting nginx and other infrastructure-as-code files. Brilliant project. Thanks again! 👍

Reply:
Awesome! Glad to hear you learned a few things! And thanks for the contribution and the praise 😄 

Looking forward to those new PRs. After all, the changes we made were meant to make the project able to support those kind of files more easily, so I'm eager to see if we got it right.